idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -735,12 +774,19 @@ class SkipGsnap(Exception):
 
 
 def run_gsnap_filter(input_fas, uploader_start):
+    """GSNAP is an aligner used for filtering out host genomes. Two input
```

Comment:
Would change to "GSNAP is an aligner we use to filter out reads mapping to the host genome."
Would also add why we do this additional step after STAR/bowtie-based filtering: to increase our sensitivity for host filtering. We currently only do this for the human host -- we use the chimp reference genome as an extension to increase our sensitivity for human matches